201-vmss-automation-dsc/azuredeploy.json

```diff
@@ -2,642 +2,286 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "domainNamePrefix": {
-      "type": "string",
-      "metadata": {
-        "description": "The public DNS name label for the service.  The pattern will follow: <domainNamePrefix>.<region>.cloudapp.azure.com"
-      }
-    },
-    "vmSku": {
-      "type": "string",
-      "defaultValue": "Standard_A2",
-      "metadata": {
-        "description": "VM size for the VM Scale Set + the mgmt VM"
-      }
-    },
-    "windowsOSVersion": {
+    "location": {
       "type": "string",
-      "defaultValue": "2012-R2-Datacenter",
       "metadata": {
-        "description": "The Windows Server version for the VM. This will pick a fully patched image of this given Windows version"
-      }
+        "description": "The Azure location to deploy all resources"
+      },
+      "allowedValues": [
+        "japaneast",
+        "eastus2",
+        "westeurope",
+        "southeastasia",
+        "southcentralus",
+        "australiaeast",
+        "koreacentral",
+        "westus2",
+        "brazilsouth",
+        "uksouth",
+        "westcentralus",
+        "northeurope",
+        "canadacentral",
+        "australiasoutheast",
+        "centralindia",
+        "francecentral",
+        "centraluseuap"
```

Comment:
remove this one since most users will not have access to it (and deployment will fail)

Reply:
I thought this is what you were asking for when you said a validated list of locations.  Did I not understand where to put it?

Reply:
@bmoore-msft

Reply:
Sorry, specifically remove the "centraluseuap" array element.  The rest are fine/correct.

Reply:
Got it.  Sorry I didn't catch the line reference.  Change made.